hub-js/graphql/public/schema.graphql

```diff
@@ -235,7 +235,7 @@ type InterfaceInput @additionalLabels(labels: ["published"]){
 
 type InputParameter @additionalLabels(labels: ["published"]){
   name: String!
-  jsonSchema: Any
+  typeRef: TypeReference! @relation(name: "OF_TYPE", direction: "OUT")
```

Comment:
The `jsonSchema` cannot be removed

see:
- https://github.com/capactio/capact/pull/402#discussion_r675419435
- https://structsure.slack.com/archives/C01DTR12JUA/p1627031890062100?thread_ts=1626961877.057900&cid=C01DTR12JUA

TL;DR; we already use it and thanks to that we also reduce the boilerplate.